biz.aQute.bndlib/src/aQute/lib/deployer/FileRepo.java

```diff
@@ -252,6 +259,7 @@ interface Config {
 	String									name;
 	boolean									inited;
 	boolean									trace;
+	String									tags;
```

Comment:
I'd make this Set<String>.

I am leaning to make this a domain object, like `Tags` but there are two forces. Tags membership testing could be highly optimized and maybe one day we need logic operators like union and subset. It is also more efficient since you now parse the string every time someone calls getTags. 

Then again, Set<String> is very close to the concept of this object and is standard. And then we could make Tags implement Set<String> ... Tough :-)

Anyway, I do not think we should keep the raw string that would require parsing all the time.

Reply:
I made it the new Tags class which is a Set.